Dockerfile

```diff
@@ -4,6 +4,7 @@ LABEL maintainer="thnolan@redhat.com"
 LABEL name="raincatcher-keycloak-docker"
 LABEL version="1.0.0"
 
+# Enviornment Variables
 ENV KEYCLOAK_VERSION 3.1.0.Final
 ENV KEYCLOAK_ADMIN_USERNAME admin
 ENV KEYCLOAK_ADMIN_PASSWORD Password1
```

Comment:
@twnolan Can we update the admin's password to `admin` too? Its the one I typically use and think its the one used in the keycloak docs.